metricbeat/docs/fields.asciidoc

```diff
@@ -62292,6 +62292,60 @@ format: bytes
 
 --
 
+[float]
+=== network
+
+Network-specific metrics
+
+
+[float]
+=== incoming
+
+Incoming network counters
+
+
+*`system.process.network.incoming.tcp`*::
```

Comment:
We should be using the best matching field from ECS for new fields. 

In this case I would follow https://www.elastic.co/guide/en/ecs/current/ecs-network.html#field-network-direction